src/events/deposit.service.ts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Injectable } from '@nestjs/common';
+import { Deposit } from '@prisma/client';
+import { ApiConfigService } from '../api-config/api-config.service';
+import { SortOrder } from '../common/enums/sort-order';
+import { PrismaService } from '../prisma/prisma.service';
+
+@Injectable()
+export class DepositService {
```

Comment:
Same nit as the controller
```suggestion
export class DepositsService {
```

Reply:
done